.gitignore

```diff
@@ -0,0 +1,4 @@
+**.plan
```

Comment:
I believe for this to be parsed correctly you have to have a slash after the double asterisks, like so:
```
**/.plan
**/private.tfvars
**/.terraform
**/*.backup
```

Reply:
I don't think this matters as it was ignoring without out it, but I'll change.